python/servo/build_commands.py

```diff
@@ -423,7 +362,36 @@ def package_gstreamer_dylibs(cross_compilation_target, servo_bin):
     return True
 
 
-def package_gstreamer_dlls(env, servo_exe_dir, target):
+def copy_windows_dlls_to_build_directory(servo_binary: str, target_triple: str):
+    servo_exe_dir = os.path.dirname(servo_binary)
+    assert os.path.exists(servo_exe_dir)
+
+    build_path = path.join(servo_exe_dir, "build")
+    assert os.path.exists(build_path)
+
+    # Copy in the built EGL and GLES libraries from where they were built to
+    # the final build dirctory
+    def find_and_copy_built_dll(dll_name):
+        try:
+            file_to_copy = next(pathlib.Path(build_path).rglob(dll_name))
+            shutil.copy(file_to_copy, servo_exe_dir)
+        except StopIteration:
+            print(f"WARNING: could not find {dll_name}")
+
+    print(" • Copying ANGLE DLLs to binary directory...")
+    find_and_copy_built_dll("libEGL.dll")
+    find_and_copy_built_dll("libGLESv2.dll")
+
+    print(" • Copying GStreamer DLLs to binary directory...")
+    if not package_gstreamer_dlls(servo_exe_dir, target_triple):
+        return False
```

Comment:
We return False here, but the caller doesn't seem to check this return value. Also the type of this function doesn't specify that we are returning a boolean.

Reply:
I've marked this method as returning a boolean and properly handle the result in the caller now.